src/commands/object/delete.ts

```diff
@@ -30,13 +33,29 @@ export class DeleteCommand extends Command {
                 console.log();
             }
         } else {
-
             const objects = session.find({id:Buffer.from(params.oid,'hex')});
-            session.destroy(objects.items(0)!);
-            session.destroy(objects.items(1)!);
-            console.log('Object deleted.');
+            if (!objects) {
+                throw new Error(`Object by ID '${params.oid}' is not found`);
+            }
+            if(this.parent!.parent instanceof Dynamic){
```

Comment:
Not a great way to check this. It defeats the composability that currently exists when defining commands.